index.js

```diff
@@ -0,0 +1,91 @@
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+openNav();
+closeNav();
+const section2 = {
+  images: ['assets/solomon.png', 'assets/haftamu.jpg', 'assets/tsegay.png', 'assets/kinf.png', 'assets/berihu.png', 'assets/haftamu-m.png'],
+  trainers: ['Solomon Gebereslassie', 'Haftamu Desta', 'Tseegay Kidu', 'Kinfe Gbrekirstos', 'Berihu Tesfay', 'Haftamu Mokenen'],
+  title: ['substation construction expert', 'Test and commissioning expert', 'Railway Engineer', 'Transmmission construction expert', 'Substation design expert', 'Transmmission substation director'],
+  resume: ['Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus', 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus', 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus', 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus', 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus', 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus'],
+};
+const detailtrainers = `<div class="center">
```

Comment:
- [ ] Kindly make this variable holds HTML dynamically because it makes it easy for you to add more HTML to display
You can do something like the following:

```JavaScript
window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })
}
```